tests/test_awslambda/test_lambda.py

```diff
@@ -151,6 +151,34 @@ def test_invoke_event_function():
     json.loads(success_result["Payload"].read().decode("utf-8")).should.equal({})
 
 
+@mock_lambda
+def test_invoke_dryrun_function():
+    conn = boto3.client("lambda", "us-west-2")
```

Comment:
Should we be declaring this in a constructor or up around line 32 so that it can be re-used? Also don't like that the region is hardcoded (as it is in several places in this file) when it's being declared and initialized as `_lambda_region" up top

Reply:
Fair point for the _lambda_region. Didn't realize that it was already declared up top. I've changed it in this test, and have changed it everywhere else in a separate PR

For the constructor - it has to be created inside the mock_lambda decorator for the mock to work, as far as I'm aware, so it's not as easy as making it a global variable